deliver/lib/deliver/options.rb

```diff
@@ -10,7 +10,7 @@ def self.available_options
       [
         FastlaneCore::ConfigItem.new(key: :username,
                                      short_option: "-u",
-                                     env_name: "DELIVER_USERNAME",
+                                     env_name: "DELIVER_USER",
```

Comment:
This is a breaking change, we'd have to support both parameter. The easiest way to do so is to change the default value to

```
default_value: user || ENV["DELIVER_USER"]
```